doc/mgr/progress.rst

```diff
@@ -32,27 +32,27 @@ Show the summary of all the ongoing and completed events and their duration:
 
 .. prompt:: bash #
 
-  ceph progress
+   ceph progress
 
 Show the summary of ongoing and completed events in JSON format:
 
 .. prompt:: bash #
 
-  ceph progress json
+   ceph progress json
 
 Clear all ongoing and completed events:
 
 .. prompt:: bash #
 
-  ceph progress clear
+   ceph progress clear
 
 PG Recovery Event
 -----------------
 
-An event for each PG affected by recovery event can be shown in
-`ceph progress` This is completely optional, and disabled by default
-due to CPU overhead:
+An event for each PG affected by recovery event can be shown in ``ceph
+progress``. This is completely optional, and disabled by default due to CPU
```

Comment:
As opposed to incompletely? Would elide the adverb.

Reply:
https://github.com/ceph/ceph/pull/63633